main/lang/catalan/trad4all.inc.php

```diff
@@ -7986,4 +7986,9 @@
 $DesignWithBadgeStudio = "Dissenyar amb Badge Studio";
 $DesignWithBadgeStudioComment = "Utilitzi Badge Studio per crear la seva pròpia insígnia dins de la seva plataforma";
 $YouAlreadySentThisFile = "Ja va enviar aquest o un altre arxiu amb el mateix nom. Si us plau assegureu-vos d'enviar cada arxiu una sola vegada.";
+$StudentCourseProgress = "Progrés: %s&#x00025;";
+$StudentCourseScore = "Puntuació: %s&#x00025;";
+$StudentCourseCertificate = "Certificat: %s";
+$MyCourseProgressTitle = "Mostra progrés del curs a l'estudiant";
+$MyCourseProgressTemplateComment = "En activar aquesta opció se'ls mostraran als estudiants el progrés dels seus cursos a la pàgina 'Els meus cursos'.";
```

Comment:
Language files cannot be modified directly through a PR. You have to go through official developers or have admin status in translate.chamilo.org to add new variables.
When a language variable includes a %s sign, we name the variable with an uppercase "X" so that it is clear where the %s part is.
In this case, I have created the following variables for you (you will still need to connect to translate.chamilo.org to add the Catalan translations):
- $StudentCourseProgressX
- $StudentCourseScoreX
- $StudentCourseCertificateX
The two last variables should disappear for now as they won't be used until we move to the next major version and add these options to the settings_current table.
Please update your code to add the "X" letter at the end of your language variables.